common/changes/@typespec/versioning/revert1647_2023-03-01-22-12.json

```diff
@@ -2,7 +2,7 @@
   "changes": [
     {
       "packageName": "@typespec/versioning",
-      "comment": "Support having a shared versioning enum that is declared outside the versioned namespace.",
+      "comment": "Revert PR #1647.",
```

Comment:
```suggestion
      "comment": "",
```

No changelog needed since this is reverted before ever shipping.

Reply:
The CI complained until I added a changelog